supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesCallActor.scala

```diff
@@ -0,0 +1,81 @@
+package cromwell.backend.impl.jes
+
+import java.nio.file.Path
+
+import akka.actor.ActorRef
+import better.files._
+import cromwell.backend.BackendJobLifecycleActor
+import cromwell.backend.impl.jes.JesAsyncBackendJobExecutionActor.WorkflowOptionKeys
+import cromwell.backend.impl.jes.io.{JesAttachedDisk, JesWorkingDisk}
+
+import scala.language.postfixOps
+
+trait JesCallActor extends BackendJobLifecycleActor {
```

Comment:
ToL: This isn't so much an Actor as a bundle of information. If it's not a massive refactor, is it possible to turn this into a mixin along the lines of
```
trait JesCallInformations { this: BackendJobLifecycleActor =>
```